Submitted By:            Igor Živković
Date:                    2014-10-17
Initial Package Version: 0.74
Upstream Status:         Fixed
Origin:                  Upstream
Description:             Fixes build with gcc version >= 4.6.

diff -Naur dosbox-0.74.orig/include/dos_inc.h dosbox-0.74/include/dos_inc.h
--- dosbox-0.74.orig/include/dos_inc.h	2010-05-10 19:43:54.000000000 +0200
+++ dosbox-0.74/include/dos_inc.h	2014-10-17 21:44:03.666520211 +0200
@@ -28,6 +28,8 @@
 #include "mem.h"
 #endif
 
+#include <stddef.h>
+
 #ifdef _MSC_VER
 #pragma pack (1)
 #endif
